modules/carto/package.json

```diff
@@ -43,6 +43,7 @@
     "d3-color": "^2.0.0",
     "d3-format": "^2.0.0",
     "d3-scale": "^3.2.3",
+    "h3-js": "^3.6.0",
```

Comment:
I think adding h3 as a full dependency will make the bundle considerable bigger.

Reply:
I think this will prevent it from being bundled, like is currently the case in `geo-layers`

https://github.com/visgl/deck.gl/blob/master/scripts/bundle.config.js#L18-L21

Reply:
`dependencies` in `package.json` doesn't affect bundle size, it's webpack config and as @felixpalmer pointed out, h3-js is already configured as external dependency.

This patch only "sanitizes" dependencies as _we_ truly depend h3-js and it's good practice not to rely on transitive dependencies (from geo-layers in this case), also ... eslint complains about using deps not specified in package.json.